plugin/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -638,7 +639,7 @@ private CpsScript parseScript() throws IOException {
             for (Entry<String, String> e : loadedScripts.entrySet()) {
                 shell.reparse(e.getKey(), e.getValue());
             }
-        } catch (groovyjarjarasm.asm.MethodTooLargeException x) {
+        } catch (MethodTooLargeException x) {
             LOGGER.log(Level.SEVERE, "FAILED to parse WorkflowScript (the pipeline script) due to MethodTooLargeException: " + x.toString());
```

Comment:
```suggestion
```

as per https://github.com/jenkinsci/workflow-cps-plugin/pull/817/files/c438ebd0d6071eadd7fea8cd096dc5e6a51dde69..28cb0c3e54468cc8b8f527358d4c8d74b8fd9ace#r1408121159

Reply:
I guess this got outdated by the latest commit?